packages/clerk-js/src/ui/components/UserProfile/ConnectedAccountsSection.tsx

```diff
@@ -164,7 +164,7 @@ const ConnectedAccount = ({ account }: { account: ExternalAccountResource }) =>
           <ImageOrInitial />
           <Box sx={{ whiteSpace: 'nowrap', overflow: 'hidden' }}>
             <Flex
-              gap={2}
+              gap={1}
```

Comment:
This fixes an issue with the spacing between the provider and email address:

With gap 1:

![CleanShot 2024-12-06 at 07 38 19](https://github.com/user-attachments/assets/94fc61cf-3757-4ca4-946e-f31981f7172e)

With gap 2:

![CleanShot 2024-12-06 at 07 37 55](https://github.com/user-attachments/assets/4a67ae27-49a9-461d-b224-c15691f8da97)